ui/packages/consul-ui/app/components/hashicorp-consul/index.scss

```diff
@@ -2,6 +2,12 @@
   nav .dcs {
     @extend %main-nav-vertical-hoisted;
     left: 100px;
+
+    .dcs-message {
+      padding: 8px 12px;
+      border-bottom: 1px solid rgb(var(--tone-gray-400));
+      max-width: fit-content;
+    }
```

Comment:
Self-note: We used to do this elsewhere, so we might have something that already exists for this, I'm gonna check in case. This one is not a blocker at all, just wanted to check in a bit.